maestro-sdk/ios/Maestro SDK/Maestro SDK/MaestroMockServerSdk.swift

```diff
@@ -7,7 +7,14 @@ public class MaestroMockServerSdk {
             fatalError("projectId is not initialized. Did you call MaestroSdk.setup()?")
         }
         
-        let sessionId = NSUUID().uuidString
+        let sessionIdFromEnvVar = ProcessInfo.processInfo.environment["MAESTRO_SESSION_ID"]
+
+        var sessionId = NSUUID().uuidString
+        if (sessionIdFromEnvVar != nil) {
+            if let uuid = UUID(uuidString: sessionIdFromEnvVar!) {
+                sessionId = uuid.uuidString
+            }
+        }
```

Comment:
This can be written more swift-y:
```
let sessionId: String
if let sessionIdFromEnvVar = ProcessInfo.processInfo.environment["MAESTRO_SESSION_ID"],
    let uuid = UUID(uuidString: sessionIdFromEnvVar) {
    sessionId = uuid.uuidString
} else {
    sessionId = UUID().uuidString
}
```